core/src/main/resources/jenkins/security/ApiTokenProperty/config.jelly

```diff
@@ -127,6 +127,7 @@ THE SOFTWARE.
                             <span class="token-cancel">
                                 <f:repeatableDeleteButton value="${%Cancel}" />
                             </span>
+                            <l:copyButton message="${%Copied}" text="" clazz="invisible" style="margin-left: 6px;"/>
```

Comment:
nit: use css style instead of inline ones

As you need to modify the CSS, take oppportunity to also add the `margin-left: 6px` to `.token-list .token-list-new-item .copy-button` new "category".

Reply:
Good point.